README.md

```diff
@@ -1,15 +1,14 @@
 # JavaScript Math
 
-## Overview
+In Ruby solving for the square root of a number involves using a class method `.sqrt` on the `Math` class. While JavaScript supports a lot of the same mathematical operations as Ruby, there is one big difference. `Math` in JavaScript is actually a single built-in object that you can perform operations on. This `Math` object does have plenty of convenience methods that allow you to do math simply and easily. You can read more about math in JavaScript [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math).
```

Comment:
I'm a little bit confused here- what's the difference between the `Math` class in Ruby and the `Math` object in Javascript?